templates/account/password_set.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% load bootstrap4 %}
+{% load i18n %}
+
+{% block content %}
+
+<h1>{% trans 'Your profile settings' %}</h1>
+{% include 'users/userprofilenav.html' %}
+<div class="px-2 pt-5">
```

Comment:
Vllt sollte man hier noch irgendwo sowas wie
You are can set your password here so you can log in to trainhorizon also without your github account?
dazu schreiben?